hw/top_earlgrey/ip/ast/doc/_index.md

```diff
@@ -25,772 +22,875 @@ shown in the diagram, but will be explained as well.
 
-  -  Clock signals start with clk\_\*
+-   Clock signals start with clk\_\*
 
-  -  Inputs and outputs are marked with \*\_i/\*\_o
+-   Inputs and outputs are marked with \*\_i/\*\_o
 
-  -  Analog signals are marked with \*\_a
+-   Analog signals are marked with \*\_a
 
-  -  Non-core level signals are marked with \*\_h
+-   Non-core level signals are marked with \*\_h
 
-  -  Dual and negative polarity signals are marked with \*\_p/n
+-   Dual and negative polarity signals are marked with \*\_p/n
 
 ### Clock domains column
 
-  -  sys - system clock, mainly used for high performance and security
-     modules. Up to 100MHz
+-   sys - system clock, mainly used for high performance and security
+    > modules. Up to 100MHz
 
-  -  io - peripheral clock source, mainly used for peripherals and I/O
-     related functionality. Up to 96MHz (divided by 4 by the clock
-     manager)
+-   io - peripheral clock source, mainly used for peripherals and I/O
+    > related functionality. Up to 96MHz (divided by 4 by the clock
+    > manager)
 
-  -  susb - USB module source clock. 48MHz
+-   usb - USB module source clock. 48MHz
 
-  -  aon - Always-on domain clock. The only active clock while chip is
-     in deep-sleep power state, 200KHz
+-   aon - Always-on domain clock. The only active clock while chip is in
+    > deep-sleep power state, 200KHz
 
-  -  async - when listed as async, it means it does not matter what
-     domain drives the signal
+-   async - when listed as async, it means it does not matter what
+    > domain drives the signal
 
-  -  Input clocks: Each functional interface has a dedicated clock
-     named after the interface.
+-   Input clocks: Each functional interface has a dedicated clock named
+    > after the interface.
 
 <table>
+<colgroup>
+<col style="width: 20%" />
+<col style="width: 4%" />
+<col style="width: 7%" />
+<col style="width: 7%" />
+<col style="width: 59%" />
+</colgroup>
 <thead>
 <tr class="header">
 <th><strong>Signal Name &amp; Affiliation</strong></th>
 <th><strong>I/O</strong></th>
-<th><strong>Width</strong></th>
+<th><p><strong>Width/</strong></p>
+<p><strong>Type/</strong></p>
+<p><strong>Struct</strong></p></th>
 <th><strong>Clock Domain</strong></th>
 <th><strong>Description</strong></th>
 </tr>
-</thead>
-<tbody>
 <tr class="odd">
-<td><strong>Power Supplies</strong></td>
-<td></td>
-<td></td>
-<td></td>
-<td></td>
-</tr>
-<tr class="even">
-<td>VCC</td>
-<td>I</td>
-<td></td>
-<td></td>
-<td><p>VCC is the main power supply. It is driven from an external source and is used to power the internal VCMAIN and VCAON power domains.</p>
-<p>VCC must always be present when the device is functioning; VCC is also used to power a number of pads that must be always on when the device is functioning.</p></td>
+<th colspan="5"><strong>Power Supplies</strong></th>
+</tr>
+<tr class="header">
+<th>VCC</th>
+<th>I</th>
+<th></th>
+<th></th>
+<th><p>VCC is the main power supply. It is driven from an external

```

Comment:
Is there a reason why all these cells have been converted from `<td>` (table data) to `th` (table header)?
It causes all cells to be rendered in bold and centered text, which does not look great IMO, see here:
![image](https://user-images.githubusercontent.com/41358501/192407177-88b3ed66-4491-400d-9f4a-9b3378a1a29a.png)
this is how it looks with `td`:
![image](https://user-images.githubusercontent.com/41358501/192407246-fc040dd5-5de3-4369-a399-800d100b68aa.png)